plugins/module_utils/foreman_helper.py

```diff
@@ -816,7 +827,11 @@ def _lookup_entity(self, identifier, entity_spec, params=None):
         try:
             if 'scope' in entity_spec:
                 for scope in entity_spec['scope']:
-                    params.update(self.scope_for(scope))
+                    if resource_type in ['content_views', 'repositories'] and scope == 'lifecycle_environment':
```

Comment:
this is horrible, but Katello keeps on calling LCEs "environments" in so many places…

Reply:
https://projects.theforeman.org/issues/31714